docs/examples/301/log-analytics-with-datasources-solutions/main.bicep

```diff
@@ -0,0 +1,102 @@
+@description('Location of the workspace')
+param location string = resourceGroup().location
+@description('Name of the workspace')
+param name string
+@allowed([
+  'PerGB2018'
+  'Free'
+  'Standalone'
+  'PerNode'
+  'Standard'
+  'Premium'
+])
+@description('Sku of the workspace')
+param sku string
+@description('The workspace data retention in days, between 30 and 730')
+@minValue(7)
+@maxValue(730)
+param retentionInDays int
+@description('Solutions to add to workspace')
+param solutions array = []
+@description('Name of automation account to link to workspace')
+param automationAccountName string = ''
+@description('Datasources to add to workspace')
+param dataSources array = []
+@description('Enable lock to prevent accidental deletion')
+param enableDeleteLock bool = false
+@description('Enable diagnostic logs')
+param enableDiagnostics bool = false
+@description('Storage account name. Only required if enableDiagnostics is set to true.')
+param diagnosticStorageAccountName string = ''
+@description('Storage account resourceop. Only required if enableDiagnostics is set to true.')

```

Comment:
nit - looks like this should read: `Storage account resource group.`

Reply:
done